opening_counter.py

```diff
@@ -8,20 +8,8 @@
 import chess.pgn
 
 parser = argparse.ArgumentParser(description=__doc__)
```

Comment:
You can delete this file,  it was an experiment and I don't think it's add value to the project.

Reply:
Done